### PR TITLE
[#98558456] Add pricing fields to draft service editing

### PR DESCRIPTION
**Pull request by robyoung**
https://www.pivotaltracker.com/story/show/98558456

- Use the pricing form template from the frontend toolkit to render the pricing fields.
- Parse the pricing fields into the correct fields for the data API
- Render pricing specific error messages
- Render a human readable price string on the service edit page

screen shot 2015-08-03 at 17 29 07

screen shot 2015-08-03 at 17 29 19

## Depends on
- [x] https://github.com/alphagov/digitalmarketplace-utils/pull/114
- [ ] https://github.com/alphagov/digitalmarketplace-api/pull/195

### CODE REVIEW

File: tests/app/main/test_services.py [hunk 1]
```diff
@@ -645,3 +688,41 @@ def test_update_non_existent_draft_section_returns_404(self, data_api_client):
...
+        data_api_client.get_draft_service.return_value = self.draft_service
+        res = self.client.get('/suppliers/submission/services/1')
+        document = html.fromstring(res.get_data(as_text=True))
+
+        assert_equal(res.status_code, 200)
+        service_price_row_xpath = '//tr[contains(.//span/text(), "Service price")]'
+        service_price_xpath = service_price_row_xpath + '/td[@class="summary-item-field"]/span/text()'
+        assert_equal(
+            document.xpath(service_price_xpath)[0].strip(),
+            "£12.50 to £15 per Person per Second")
```
Comment (pcraig3) [inline tests/app/main/test_services.py]: Seems like:
- [ ] a good deal
- [x] not a good deal

Comment (robyoung) [inline tests/app/main/test_services.py]: Good spot on the unicode bug.

**Comment by TheDoubleK** [comment]
:+1: This all looks good to me. Good luck with the conflict resolution... I think @quis has made some changes to the formatting data bits of `services.py` too - not sure if his stuff is in yet, but might be worth taking a look together to check you're not stepping on each other's toes too much.

**Comment by TheDoubleK** [comment]
BTW - that "Depends on" bit in the pull-request message is sweet.  Does Github automatically tick the boxes as they get merged?  Or do you have to do it manually? (I noticed you get a nice progress bar in the pull-request list too)

**Comment by robyoung** [comment]
That @TheDoubleK I'm pretty much rewriting it again on top of current master :(

**Comment by TheDoubleK** [comment]
Merge, don't rebase when it's that bad.  Still no fun though.

**Comment by robyoung** [comment]
Closing in favour of #144

**Pull request closed without merging**